addons/medical_engine/XEH_postInit.sqf

```diff
@@ -1,7 +1,11 @@
 #include "script_component.hpp"
 
 ["CBA_settingsInitialized", {
-    if !(GETEGVAR(medical,enabled,false)) exitWith {};
+    if !(GETEGVAR(medical,enabled,false)) exitWith {
+        // Call manually (see CfgFunctions.hpp)
+        // https://community.bistudio.com/wiki/Arma_3:_Functions_Library#Config_Levels
+        ["postInit", didJIP] spawn BIS_fnc_reviveInit;
```

Comment:
I don't know, it's something to investigate. SOG doesn't seem to set the function to not run in postInit, but it might do something somewhere else.
As it is, it's probably a bad idea, so definitely will need to find a better solution.